packages/tentales-example-advanced/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "../../tsconfig.json",
```

Comment:
Should this example really extend the root tsconfig? shouldn't it be more stand-alone? 🤔

Reply:
It needs the `paths` from it.